cfme/common/vm_views.py

```diff
@@ -105,7 +105,11 @@ def data(self):
                 return {}
 
             try:
-                state = quad_data.xpath(self.QUADRANT.format(pos="b"))[0].get('style')
+                if self.browser.product_version == 'master':
+                    state = quad_data.xpath(self.QUADRANT.format(pos="b"))[0].get('src')
```

Comment:
REQUIRED:
please replace these two commands with one command and different properties
```
state_property = 'src' if self.browser.product_version == 'master' else 'style'
state = quad_data.xpath(self.QUADRANT.format(pos="b"))[0].get(state_property)
```